src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -117,35 +117,38 @@ public void processHearingWrapper(HearingWrapper wrapper)
 
     private void createHearing(HearingWrapper wrapper) throws UpdateCaseException, ListingException {
         SscsCaseData caseData = wrapper.getCaseData();
-
         String caseId = caseData.getCcdCaseId();
         HearingsGetResponse hearingsGetResponse = hmcHearingApiService.getHearingsRequest(caseId, null);
-        CaseHearing hearing = HearingsServiceHelper.findExistingRequestedHearings(hearingsGetResponse, false);
-        HmcUpdateResponse hmcUpdateResponse;
+        CaseHearing hearing = HearingsServiceHelper.findExistingRequestedHearings(hearingsGetResponse, true);
         overridesMapping.setDefaultListingValues(wrapper.getCaseData(), refData);
 
-        if (isNull(hearing)) {
-            HearingRequestPayload hearingPayload = hearingsMapping.buildHearingPayload(wrapper, refData);
-            log.debug("Sending Create Hearing Request for Case ID {}", caseId);
-            hmcUpdateResponse = hmcHearingApiService.sendCreateHearingRequest(hearingPayload);
-
-            var johTiers = hearingPayload.getHearingDetails().getPanelRequirements().getRoleTypes();
-            log.info("Saving JOH tiers ({}) onto the case ({})", johTiers, caseId);
-            wrapper.getCaseData().setPanelMemberComposition(new PanelMemberComposition(johTiers));
-
-            log.debug("Received Create Hearing Request Response for Case ID {}, Hearing State {} and Response:\n{}",
-                    caseId, wrapper.getHearingState().getState(), hmcUpdateResponse.toString());
-        } else {
-            hmcUpdateResponse = HmcUpdateResponse.builder()
-                    .hearingRequestId(hearing.getHearingId())
-                    .versionNumber(getHearingVersionNumber(hearing))
-                    .status(hearing.getHmcStatus())
+        if (nonNull(hearing)) {
```

Comment:
are we changing the behaviour so that we always cancel the existing hearing and create a new one automatically?